nengo_dl/tests/test_transforms.py

```diff
@@ -4,66 +4,67 @@
 import numpy as np
 import pytest
 import tensorflow as tf
+from nengo.builder.transforms import ConvInc
 
+from nengo_dl.compat import ConvolutionTranspose, ConvTransposeInc, is_dummy_type
 from nengo_dl.utils import tf_gpu_installed
 
 
+@pytest.mark.parametrize("force_cpu", (False, True))
 @pytest.mark.parametrize("channels_last", (True, False))
-def test_merge_conv(Simulator, channels_last, seed, pytestconfig):
-    from nengo.builder.transforms import (  # pylint: disable=import-outside-toplevel
-        ConvInc,
-    )
+@pytest.mark.parametrize("transpose", (True, False))
+def test_merge_conv(Simulator, transpose, channels_last, force_cpu, seed, pytestconfig):
+    if transpose and is_dummy_type(ConvolutionTranspose):
+        pytest.skip("Nengo version does not support ConvolutionTranspose")
+
+    is_cpu = not tf_gpu_installed or pytestconfig.getoption("--device") == "/cpu:0"
```

Comment:
I'd keep the more general `cpu in device.lower()` approach; there are lots of valid device strings (if we wanted to be fancy we could actually build the https://www.tensorflow.org/api_docs/python/tf/DeviceSpec and extract the device from that, but that's probably overkill for a test).